### PR TITLE
Fix issues with July 2020 Patch

### PR DESCRIPTION
**Pull request by alimulhuda**
Addressed the issues identified in #103 

Tried it with the expansions enabled (RF and GS). The notification panel seems to be working and the icons in the production panel look okay (This one I would like another pair eyes to double check the visuals).

### CODE REVIEW

**Comment by Infixo** [comment]
There are changes in .lua also to utilize new _combo textures. I don’t see them here.

**Comment by alimulhuda** [comment]
If you are referring to changes in ProductionPanel.lua in unmodded game, then I don't think it impacts CQUI. I couldn't find a place where we directly use the textures. I'll take a closer look after work today.

I also think something might be wrong with EspionageChooser, I was looking over the diff and it looks like they added a function parameter to OnMissionSelected and EspionageChooser_ShowMissionBriefing. I need to start work now, but I'll take a look after.

**Comment by Infixo** [comment]
Yes, ProductinPanel.lua

It was:
local TEXTURE_BASE				:string = "UnitFlagBase";
local TEXTURE_CIVILIAN			:string = "UnitFlagCivilian";
local TEXTURE_RELIGION			:string = "UnitFlagReligion";
local TEXTURE_EMBARK			:string = "UnitFlagEmbark";
local TEXTURE_FORTIFY			:string = "UnitFlagFortify";
local TEXTURE_NAVAL				:string = "UnitFlagNaval";
local TEXTURE_SUPPORT			:string = "UnitFlagSupport";
local TEXTURE_TRADE				:string = "UnitFlagTrade";

Now it is (line 24-31)
local TEXTURE_BASE				:string = "UnitFlagBase_Combo";
local TEXTURE_CIVILIAN			:string = "UnitFlagCivilian_Combo";
local TEXTURE_RELIGION			:string = "UnitFlagReligion_Combo";
local TEXTURE_EMBARK			:string = "UnitFlagEmbark_Combo";
local TEXTURE_FORTIFY			:string = "UnitFlagFortify_Combo";
local TEXTURE_NAVAL				:string = "UnitFlagNaval_Combo";
local TEXTURE_SUPPORT			:string = "UnitFlagSupport_Combo";
local TEXTURE_TRADE				:string = "UnitFlagTrade_Combo";


And e.g.
			unitListing.FlagBase:SetTexture(textureName);
			unitListing.FlagBaseOutline:SetTexture(textureName);
			unitListing.FlagBaseDarken:SetTexture(textureName);
			unitListing.FlagBaseLighten:SetTexture(textureName);

is changed into  (line 1329)
			unitListing.FlagBase:SetTexture( textureName );

Basically all changes in .xml have corresponding changes in .lua because there is a new texture used and some ui controls are removed.
Maybe it is not valid in the heavily modded version, because it is already been changed.

**Comment by alimulhuda** [comment]
Saw some more things that needed updating/merging. The two noticeable ones are
- The religion screen in city overview panel has a colour key for religions
- The espionage mission accept screen is now fixed

**Comment by alimulhuda** [comment]
I double checked the production panel lua file and didn't notice any place where we would need to update things.

Also thought marking it for re-review would remove the old approval, but I guess not. So once someone has reviewed it with all the changes in place now, I'll merge it to main branch.
